go.mod

```diff
@@ -11,8 +11,8 @@ require (
 	github.com/go-pkgz/rest v1.16.0
 	github.com/golang-jwt/jwt v3.2.2+incompatible
 	github.com/golang-jwt/jwt/v4 v4.4.1
-	github.com/microcosm-cc/bluemonday v1.0.20
 	github.com/nullrocks/identicon v0.0.0-20180626043057-7875f45b0022
+	github.com/pkg/errors v0.9.1
```

Comment:
this appeared when running `go mod tidy` - seems some of the oauth changes brought it in